lensterxyz/lenster
# fix: "You can't reply to this post" not being visible in dark mode

## What does this PR do?

Hopefully, this will fix the `"You can't reply to this post"` text not being visible in dark mode according to your style guidelines and design choices.

I tested in light mode as well, and it is still visible with this change.

Fixes #1562 

## Type of change

- [X ] Bug fix (non-breaking change which fixes an issue)
- [X ] Enhancement (non-breaking small changes to existing functionality)

## How should this be tested?

To make sure the issue exists, open a gated post in dark mode and verify that you can't see the  `"You can't reply to this post"` text. Then, do the same with this PR applied and you should now be able to see the text in a greyish color.
